script/Ciel/Day 13/13_01_01_CIEL13_3B.txt

```diff
@@ -745,7 +745,7 @@ Getting rid of Roa isn't as straightforward as simply killing his reincarnation.
 [sha:b9a043f75e1c7c94faa803bda4975e102591c719]{
 -- Page 175, Offset 33685.
 -- まったく、ノエルのせいで台無しになるところでした。
-It was close; Noel very nearly sent everything right down the drain.
+It was close, Noel very nearly sent everything right down the drain.
```

Comment:
Would lean to keep this one, the two fragments really deserve the separation here.